.github/workflows/release.yml

```diff
@@ -37,43 +41,121 @@ jobs:
           python -m build --sdist
           twine check --strict dist/*
 
-      - uses: actions/upload-artifact@v3
+      - uses: actions/upload-artifact@v4
         with:
+          name: shapely-sdist
           path: ./dist/*.tar.gz
-          retention-days: 30
+          retention-days: 5
+          compression-level: 0
+
+  build_wheels_linux:
+    name: Build wheels on Linux with ${{ matrix.baseimage }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: x86_64
+            baseimage: manylinux2014_x86_64
+            basetag: 2024.08.12-1
```

Comment:
Do you anticipate this to be different across images? Otherwise we could also (or at least for now) add this as top-level matrix entry to not have to repeat it here?

Reply:
This could probably be a top-level item, as all images appear to have the same tag. It's better to tag than it is to have `latest` for stability and hopefully caching (I'm still observing how image caching works in GHA).

Reply:
moved to `env.basetag`